device/accton/x86_64-accton_as7716_32x-r0/Accton-AS7716-32X/sai.profile

```diff
@@ -0,0 +1,2 @@
+SAI_INIT_CONFIG_FILE=/etc/bcm/th-as7716-32x100G.config.bcm
```

Comment:
@stcheng, Sure!  We put it here just to reminder you that we've came up our config.bcm for you to package it.